app/display/editor/doc/widgets_properties.rst

```diff
@@ -10,9 +10,20 @@ Common Widget Properties
 The following most basic properties are available on all widgets:
  - "x", "y", "width", "height" that define its position within the display.
  - "name" that can be used to identify a widget.
+ - "tooltip", used to provide relevant information when mouse pointer hovers over the widget. See below for additional information.
 
 While not available on all widgets the following properties are associated with almost all widgets
  - "foreground_color"
  - "font" the font associated with the text of the widget
 
- - "pv-name" the name of a datasource channel which is to be monitored and the values from the channel are used to update the state of the widget
\ No newline at end of file
+ - "pv-name" the name of a datasource channel which is to be monitored and the values from the channel are used to update the state of the widget
```

Comment:
Should "pv-name" be "pv_name"?

Reply:
Granted, "PV" is maybe a bit ambiguous. I'll update.